filecoin-proofs/src/api.rs

```diff
@@ -277,7 +277,7 @@ pub fn seal<T: AsRef<Path>>(
             .into_iter()
             .map(|p| p.number_of_leaves)
             .collect::<Vec<_>>(),
-        (sector_bytes / 127) * 4,
+        sector_bytes >> 5,
```

Comment:
Note for @sidke - Before this changeset, this code had only been exercised with 1024B (bit-padded) sectors. So, by virtue of the way that division with `usize` is handled in Rust, things worked as intended (with 1024B sectors):

```
(sector_bytes / 127) * 4 == floor(1024 / 127) * 4
                         == floor(8.06299) * 4
                         == 8 * 4
                         == 32
```